build.xml

```diff
@@ -132,17 +132,76 @@
 	</target>
 
     <target name="prodVersion">
-            <!-- Full version examples: 1.0.0-SNAPSHOT-0-HASH, 1.0.0-0-HASH, 1.0.0-1-HASH, etc -->
-            <exec outputproperty="fullVersion" dir="." executable="sh">
-			<arg value="-c" />
-                <arg value="git describe --long | tr -d v" />
-		</exec>
-            <!-- Version examples: 1.0.0-SNAPSHOT, 1.0.0, etc -->
-            <exec outputproperty="releaseVersion" dir="." executable="sh">
-                <arg value="-c" />
-                <arg value="git describe --abbrev=0 | tr -d v" />
-		</exec>
-            <echo>Full Version: ${fullVersion}</echo>
-            <echo>Version: ${releaseVersion}</echo>
-	</target>
+        <!-- Full version examples: 1.0.0-SNAPSHOT-0-HASH, 1.0.0-0-HASH, 1.0.0-1-HASH, etc -->
+        <exec outputproperty="fullVersion" dir="." executable="sh">
+            <arg value="-c" />
+            <arg value="git describe --long | tr -d v" />
+        </exec>
+        <!-- Version examples: 1.0.0-SNAPSHOT, 1.0.0, etc -->
+        <exec outputproperty="releaseVersion" dir="." executable="sh">
+            <arg value="-c" />
+            <arg value="git describe --abbrev=0 | tr -d v" />
+        </exec>
+        <echo>Full Version: ${fullVersion}</echo>
+        <echo>Version: ${releaseVersion}</echo>
+    </target>
+
+    <target name="dockerImageBuild" depends="prodVersion,createBuildDir,copyAndUnzipServerUpgradeBundle,copyServerUpgradeBundle,createDockerImage" description="Copy result files to build dir">
+        <zip destfile="Build${fullVersion}/OSC-${releaseVersion}_Build${fullVersion}.zip">
+            <fileset dir="Build${fullVersion}" includes="**/*.tar **/serverUpgradeBundle-*.zip" />
+        </zip>
+    </target>
+
+    <target name="checkServerUpgradeBundleExist">
+        <available file="docker/serverUpgradeBundle*.zip" property="isServerUpgradeBundleExist"/>
+    </target>
+
+    <target name="deleteOldServerUpgradeBundle" depends="checkServerUpgradeBundleExist" if="isServerUpgradeBundleExist">
+        <delete>
+            <fileset dir="docker/" includes="serverUpgradeBundle-*.zip" />
```

Comment:
Can we have configuration items defined, where we can specify the folder names? rather than hard-coding it. Later on when we start to use Kolla Container, it would be easy to change in configuration file, rather than in the build file. also it is easier to maintain.